src/main/groovy/com/google/protobuf/gradle/GenerateProtoTask.groovy

```diff
@@ -330,12 +330,67 @@ public abstract class GenerateProtoTask extends DefaultTask {
   @Internal("Input captured by getAlternativePaths(), this is used to query alternative path by locator name.")
```

Comment:
There's some references to getAlternativePaths() here and getPluginsExecutableLocators() which look to need updating.